src/layout/IHyprLayout.cpp

```diff
@@ -215,16 +212,16 @@ void IHyprLayout::onBeginDragWindow() {
     m_vLastDragXY          = m_vBeginDragXY;
 
     // get the grab corner
-    if (m_vBeginDragXY.x < m_vBeginDragPositionXY.x + m_vBeginDragSizeXY.x / 2.0) {
-        if (m_vBeginDragXY.y < m_vBeginDragPositionXY.y + m_vBeginDragSizeXY.y / 2.0) {
+    if (m_vBeginDragXY.x < m_vBeginDragPositionXY.x + m_vBeginDragSizeXY.x / 2.f) {
```

Comment:
what is with the obsession to change .0 to .f?

Reply:
2.0 - double, 2.f - float. The compiler will complain in strict mode. But the reason is different, when I wrote the code I used 2.0 in many places, then I corrected everything together.